data/language/nl-NL.yml

```
@@ -1600,7 +1600,7 @@ strings:
   1564: "{COLOUR WINDOW_2}Intelligentie: {COLOUR BLACK}{INT16} ({STRINGID})"
   1565: "{COLOUR WINDOW_2}Agressiviteit: {COLOUR BLACK}{INT16} ({STRINGID})"
   1566: "{COLOUR WINDOW_2}Competitiveness: {COLOUR BLACK}{INT16} ({STRINGID})"
-  1567: "{SMALLFONT}{COLOUR BLACK}Toggle between single player and two player mode"
+  1567: "{SMALLFONT}{COLOUR BLACK}Wisselen tussen single player en multiplayer"
```

Ik heb een lichte voorkeur voor:
```suggestion
  1567: "{SMALLFONT}{COLOUR BLACK}Wissel tussen single player en multiplayer"
```
Akkoord?

---

Ik heb eigenlijk alle andere tooltips vertaald met de infinitief, en de knoppen ook. 

Gebiedende wijs kan wel (Apple doet dat bijvoorbeeld), maar dan is het denk ik beter om dat consequent te doen.

---

Ah ja, het is een tooltip. Prima. Ik vind het niet zo'n probleem verder, zolang het consistent is gedaan.